pyproject.toml

```diff
@@ -23,7 +23,9 @@ marshmallow = "^3.15.0"
 marshmallow-oneofschema = "^3.0.1"
 typing-extensions = "^4.3.0"
 cairo-lang = "==0.10.3"
+eth-utils = "^2.1.0"
```

Comment:
Why is this needed?

Reply:
`_starknet_keccak` uses `keccak` from eth-utils